.gitignore

```diff
@@ -266,3 +267,4 @@ $RECYCLE.BIN/
 
 # End of https://www.toptal.com/developers/gitignore/api/visualstudiocode,jupyternotebooks,python,windows,linux
 *.tflite
```

Comment:
move this to the top of file with other custom ignores

Reply:
I think that's outside the scope of this pr